pandas/util/testing.py

```diff
@@ -1172,6 +1173,13 @@ def assert_extension_array_equal(left, right):
     assert left.dtype == right.dtype
     left_na = left.isna()
     right_na = right.isna()
+
+    # TODO - maybe generate dedicated method for bitarray comparison?
```

Comment:
Comment says it all - we maybe need a dedicated method for comparing bit arrays. At the very least there's got to be a more robust way of handling this in the extension check